integrationTests/vm/txDataBuilder.go

```diff
@@ -0,0 +1,127 @@
+package vm
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+// TxDataBuilder constructs a string to be used for transaction arguments
+type TxDataBuilder struct {
+	Function  string
+	Elements  []string
+	Separator string
+}
+
+func NewTxDataBuilder() *TxDataBuilder {
+	return &TxDataBuilder{
+		Function:  "",
+		Elements:  make([]string, 0),
+		Separator: "@",
+	}
+}
+
+func (builder *TxDataBuilder) New() *TxDataBuilder {
```

Comment:
Reset? as to not confuse it with NewTxDataBuilder

Reply:
Indeed. Renamed to `Clear()`, though.